src/ui/components/map/providers/mapboxmapprovider.js

```diff
@@ -58,6 +66,10 @@ export default class MapBoxMapProvider extends MapProvider {
       center: this.getCenterMarker(mapData)
     });
 
+    this._map.addControl(new MapboxLanguage({
+      defaultLanguage: this._language
```

Comment:
I see there was some more discussion about how the fallback language should be computed. What, ultimately, was the decision there? Did we decide not to fallback?

Reply:
Rose wanted it to fall back to english and print an error in the console that the locale is not supported. That's actually the default behavior of Mapbox so I removed the fallback code I added